extensions/amp-story-auto-ads/0.1/test/test-amp-story-auto-ads.js

```diff
@@ -299,7 +301,9 @@ describes.realWin(
       });
 
       it('should create progress bar', () => {
-        const progressBar = doc.querySelector(
+        // throw new Error(doc.body.innerHTML);
```

Comment:
Debugging leftover?

Reply:
Still working on this file...it has a bunch of failing tests